core/controller/src/main/scala/org/apache/openwhisk/core/controller/Controller.scala

```diff
@@ -167,6 +167,22 @@ class Controller(val instance: ControllerInstanceId,
     }
   }
 
+  /**
+   * Handles GET /activation URI.
+   *
+   * @return running activation
+   */
+  protected[controller] val activationStatus = {
+    implicit val executionContext = actorSystem.dispatcher
+    (path("activation") & get) {
```

Comment:
would something like `/inFlightActivations` and `/inFlightActivations/instance` make more sense?

Reply:
This is based on my experience that using simple and more memorable URLs and not using case-sensitive words for APIs is better in many cases.